packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -312,7 +312,12 @@ export async function scanParentDirs(
     ]);
 
   // Priority order is Yarn > pnpm > npm > bun
```

Comment:
This comment looks like its no longer true

Reply:
True, but I'm not touching this PR and fighting flakey tests for another two days! Happy to fixup in a followup PR.

Reply:
https://github.com/vercel/vercel/pull/10603